samples/README.md

```diff
@@ -25,25 +25,32 @@ The tutorials listed below are step-by-step instructions to deploy a fully funct
-|| **Session client - SAT auth** | Connect to the MQTT broker with SAT | :yellow_circle: | :yellow_circle: | [Rust](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_mqtt/examples/sat_auth.rs) |
+| **MQTT** | **Session client** | Connect to the MQTT broker | :yellow_circle: | [.NET](/dotnet/samples/Mqtt/SessionClient) | [Rust](/rust/azure_iot_operations_mqtt/examples/simple_sample.rs) |
+|| **Session client - SAT auth** | Connect to the MQTT broker with SAT | :yellow_circle: | :yellow_circle: | [Rust](/rust/azure_iot_operations_mqtt/examples/sat_auth.rs) |
 || **Session client - x509 auth** | Connect to the MQTT broker with x509 | :yellow_circle: | :yellow_circle: | :yellow_circle: |
 ||
-| **Protocol** | **Telemetry client** | Send and receive messages to a MQTT topic | :yellow_circle: | :yellow_circle: | [Sender](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_telemetry_sender.rs)</br>[Receiver](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_telemetry_receiver.rs) |
-|| **Telemetry client with Cloud Events** | Send and receive messages to a MQTT topic with cloud events | [Go](/go/samples/protocol/cloudevents) | :yellow_circle: | [Sender](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_telemetry_sender.rs)</br>[Receiver](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_telemetry_receiver.rs) |
-|| **Command client** | Invoke and execute and command using the MQTT RPC protocol | :yellow_circle: | :yellow_circle: | [Invoker](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_rpc_invoker.rs)</br>[Executor](https://github.com/Azure/iot-operations-sdks/blob/main/rust/azure_iot_operations_protocol/examples/simple_rpc_executor.rs) |
+| **Protocol** | **Telemetry client** | Send and receive messages to a MQTT topic | :yellow_circle: | :yellow_circle: | [Sender](/rust/azure_iot_operations_protocol/examples/simple_telemetry_sender.rs)</br>[Receiver](/rust/azure_iot_operations_protocol/examples/simple_telemetry_receiver.rs) |
+|| **Telemetry client with Cloud Events** | Send and receive messages to a MQTT topic with cloud events | [Go](/go/samples/protocol/cloudevents) | [.NET](/dotnet/samples/Protocol/CloudEvents) | [Sender](/rust/azure_iot_operations_protocol/examples/simple_telemetry_sender.rs)</br>[Receiver](/rust/azure_iot_operations_protocol/examples/simple_telemetry_receiver.rs) |
+|| **Command client** | Invoke and execute and command using the MQTT RPC protocol | :yellow_circle: | :yellow_circle: | [Invoker](/rust/azure_iot_operations_protocol/examples/simple_rpc_invoker.rs)</br>[Executor](/rust/azure_iot_operations_protocol/examples/simple_rpc_executor.rs) |
```

Comment:
We've gotta have command samples for the other two languages, right? @timtay-microsoft @clecompt-msft

Reply:
From what I could tell we have them in the form of the counter sample